src/onboarding/steps.ts

```diff
@@ -243,6 +264,34 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
           }
         },
       }
+    case Screens.SignInWithEmail:
+      return {
+        next: () => {
+          if (showRecoveryPhraseEducation) {
+            navigate(Screens.AccountKeyEducation, {
+              origin: 'cabOnboarding',
+            })
+          } else if (skipVerification || numberAlreadyVerifiedCentrally) {
+            dispatch(setHasSeenVerificationNux(true))
+            finishOnboarding(Screens.ChooseYourAdventure)
+          } else {
+            // DO NOT CLEAR NAVIGATION STACK HERE - breaks restore flow on initial app open in native-stack v6
+            navigate(Screens.VerificationStartScreen)
+          }
+        },
+      }
+    case Screens.AccountKeyEducation:
+      return {
+        next: () => {
+          navigate(Screens.OnboardingRecoveryPhrase)
+        },
+      }
+    case Screens.OnboardingRecoveryPhrase:
```

Comment:
are these considered individual steps in onboarding? looking at the code in `OnboardingRecoveryPhrase`, it seems like it uses `SignInWithEmail` as the "firstScreenInCurrentStep", so I am not sure if we need these here. Wonder if we could just navigate directly from the SignInWithEmail screen, instead of considering them as a separate `step`

Reply:
Good catch we don't need these listed since we use navigation to these screens directly; fixed in a2a2d07f2712d3e3f69cfd0ee84a5fc7bdee994d!